extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/DruidK8sConstants.java

```diff
@@ -36,7 +36,8 @@
   public static final String TASK_ID_ENV = "TASK_ID";
   public static final String JAVA_OPTS = "JAVA_OPTS";
   public static final String DRUID_HOST_ENV = "druid_host";
-  public static final String DRUID_HOSTNAME_ENV = "HOSTNAME";
+  public static final String DRUID_HOSTNAME_ENV = "HOSTNAME"
```

Comment:
Did you mean to remove the semicolon here?

Reply:
oops

Reply:
should be good now